audits/multicall/202408-threat-model-multicaller.md

```diff
@@ -0,0 +1,77 @@
+# Background
+The Multicaller contract is a common pattern in the space and is used by many projects, this has some added security features that will be mentioned below.
```

Comment:
How about:

The Guarded Multi-Caller contract is a security enhanced multi-caller contract. The Multicall contract pattern in  allows multiple smart contract function calls to be grouped into a single call and the results aggregated into a single result. This provides the guarantee that function calls are executed in a single transaction atomically: they either all succeed or they all revert. Additionally, this has the benefit that it reduces the reduces the number of JSON RPC requests that need to be sent to the blockchain's RPC provider. The security enhancements offered by the Guarded Multi-caller system are described in the sections below.

Reply:
:) given the implementation, the " and the results aggregated into a single result" is not correct and needs to be removed from my suggested text.